js/layout-javascript/agenda-code.js

```diff
@@ -856,6 +856,10 @@ DynamicList.prototype.initialize = function() {
         uuid: _this.data.uuid,
         container: _this.$container,
         records: records
+      }).then(function() {
+        return Fliplet.User.getCachedSession();
```

Comment:
If "User profile data" filter type isn't used at all , then `Fliplet.User.getCachedSession()` doesn't need to be called at all.